### ISSUE TITLE
Compost

### ISSUE THREAD

**Issue by nelsonic** [dwyl/phase-three#145]
# Executive Summary

I want to produce `~3 Tons` of High Quality Vermicompost per _year_ 🏡 
for use in the `Braga` garden using _free_ organic matter. 🌱 🪴 🌳 🌴 
This is the ultimate in "circular economy" **worth at _least_ `~€1k/year`** to us. 💰 
Total cost to build what I'm thinking is `~€3k` (2 weeks). ⏳ 
So pretty quick ROI. But more importantly, a [_system_](https://github.com/dwyl/home/issues/113) everyone can use. 🕙 

# Why?

Discarding organic matter - either by burning/throwing away garden "waste" or kitchen scraps -  is like _burning_ cash! 💵 🔥 

![burning-money](https://github.com/dwyl/phase-three/assets/194400/24afb2a3-7cb5-4176-8857-54bc8f49e3eb)

_Most_ people we know that have a garden have a corresponding "green bin" where they _dispose_ of all their "waste" organic matter, giving it to the council which in turn give it to private companies which compost it and sell it back to the _same_ people with gardens ... 🤦‍♂️ 
I want to avoid this _madness_ by having a small area (though large vertical volume!) of our @home garden (70cm x 70cm x 5m) dedicated to (vermi)composting. We will place all garden waste (leaves, clippings, weeds) and kitchen scraps (peels, spoiled produce and uneaten leftovers) into the _sealed_ compost bin. The organic matter will decay and then be re-used in the garden and as "liquid feed" for the indoor plants. 

# What? 

Anyone that has done gardening / plant growing knows that plants - grown in the ground - need organic matter to thrive.
Yes, plants can be grown in hydroponic, aquaponic or even aeroponic systems that have no organic matter just liquid feed,
and we intend to explore this option: https://github.com/dwyl/learn-aquaponics/issues/2 🐟 🌱 👩‍🔬 
But for a _traditional_ garden, organic matter high in nutrients is _essential_.

# Compost

By _far_ the most effective way of recycling/reusing organic matter in your garden 
is to have a compost bin/heap where "clippings", leaves and other organic "waste" 
is stored and naturally broken down. ⏳ 

Not all compost is the same. 

## Kitchen Scraps (Peels, Cores, Seeds & Spoiled Food)

Each time I prepare food, I collect the kitchen scraps in a stainless steel bowl, e.g:

<img width="800" alt="image" src="https://github.com/dwyl/phase-three/assets/194400/7af453fd-4ce0-45d3-919d-2b1df3b39455">

Sadly, I feel like even though _I_ do my bit for composting, we still don't have a _system_.
Therefore others in the house see it as a nuisance rather than creating "black gold" from a _free_ resource. 

> Not a great photo. I didn't take it. It was sent to me as a nag/reminder to remove it from the kitchen. 
> This is why we need a clear [_system_](https://github.com/dwyl/home/issues/113) that _anyone_ can follow. 

# Vermicompost 

The "Gold Standard" in composting is using earthworms.
The worms help to break down the organic matter and by "processing" (eating & poopin') it, 
they transform it into "black gold" 

![image](https://github.com/dwyl/phase-three/assets/194400/997bb077-a849-4853-b516-3b2d76c4ede8)

If you're new to this, recommend watching this quick video: https://youtu.be/IV_kkJy3s3Q

<img width="1022" alt="image" src="https://github.com/dwyl/phase-three/assets/194400/94f63ea2-3c74-49f6-9389-9408a7f08ba2">

> Fun facts: Aristotle called earthworms "the intestines of the soil" 
and in Ancient Egypt, Cleopatra put to death _anyone_ who removed them from the soil. 
Ref: [omlet.co.uk/guide/worm_composting](https://www.omlet.co.uk/guide/worm_composting/about_worms/earthworms_through_time/)

There are commercially available compost bins such as the "Hungry Bin":
https://www.amazon.co.uk/Hungry-Bin-Flow-Through-Worm-Farm/dp/B01CRJGUOG/
<img width="1474" alt="image" src="https://github.com/dwyl/phase-three/assets/194400/ff680b2d-2c75-4700-a823-924d38c8ac39">

The downside of these, is that the plastic is relatively flimsy and decays in sunlight.
The composter I bought and setup in `Gerês` during the 2020 Covid Lockdown is already disintegrating 4 years later ... 🤦‍♂️ 

# Can't We Just _Buy_ Compost or Fertiliser? 💰 

Many people prefer to just _buy_ their organic matter. Bags of compost are sold by the _million_ to people who don't do composting ... The problem is that it's totally unsustainable! The bags are _plastic_ and the whole system is fossil fuel driven!

For reference, a "big bag" (usually `1 Ton`) of Vermicompost can be purchased for `€366` 
https://www.amazon.es/dp/B09YBHLZ7T/
<img width="1452" alt="image" src="https://github.com/dwyl/phase-three/assets/194400/154e3e52-916e-4ea5-a422-f6049d48efd8">

Searching OLX (the classifieds site) we see that someone in Aveiro has turned his composting into a side biz:
https://www.olx.pt/d/anuncio/hmus-de-minhoca-adubo-natural-IDG4Cdl.html
<img width="1310" alt="image" src="https://github.com/dwyl/phase-three/assets/194400/fd522e3d-2dce-4699-8631-4545d579863e">

But the cost per ton is still around the `€350` mark with transport to collect it.
Sooo much easier to just have a _system_ @home to make it ourselves!

# Why _Not_ to have Compost?

The only reason I can think of to _not_ have compost is the _smell_. 
But that's why _Vermicomposting_ in a sealed column is the answer!
The worms virtually eliminate the smell. And then sealing the lid of the composter will take care of the rest. 

# Todo

+ [ ] sketch design for the vertical column composter. 
+ [ ] share sketch!
+ [ ] Build it!

**Comment by iteles** [comment]
I'm 100% for the composting! It's a wonderful zero waste solution.
The only problem for me is the fruit flies invasion which lasts for the majority of the year, but which is easily solved by having a better solution within the kitchen area (sealed or in the fridge rather than out on the counter) or immediate disposal in the compost area.

As you say, we just need a system for collection and removal of the scraps.

**Comment by nelsonic** [comment]
Indeed. The only reason we end up with compost in a mixing bowl on the kitchen counter is because our _current_ compost bins (stacked plastic tubs) are _too_ many steps (10m) from the kitchen and require garden shoes to reach. 
I intend to resolve _all_ these challenges systematically.
